pkg/vendorfakes/fakehttp/handler.go

```diff
@@ -0,0 +1,77 @@
+// Code generated by counterfeiter. DO NOT EDIT.
+package fakehttp
+
+import (
+	"net/http"
+	"sync"
+)
+
+type Handler struct {
```

Comment:
Why are we generating fakes for HTTP now?

Reply:
This was from my work on the Gitlab authentication.  I created and manually moved them, but now Luiz set them up to generate automatically with everything else.